package.json

```diff
@@ -78,7 +78,7 @@
     "native-keymap": "3.3.0",
     "native-watchdog": "1.4.0",
     "node-pty": "0.11.0-beta11",
-    "spdlog": "^0.13.0",
+    "spdlog": "^0.13.6",
```

Comment:
Why was this package bumped?

Reply:
> I updated the version of spdlog used as it was throwing the following error in the console:
> 
> `Error: The module '\\?\C:\Users\t-aidaym\work\vscode\node_modules\spdlog\build\Release\spdlog.node' was compiled against a different Node.js version using NODE_MODULE_VERSION 103. This version of Node.js requires NODE_MODULE_VERSION 106`
> 
> I am wondering if I should instead revert back to 0.13.0 from 0.13.6 version of spdlog and downgrade my node version?

Did you `git clean -xfd` then run `yarn` and `yarn watch` again. We just bumped electron

Reply:
Hi, thanks for commenting. I was getting the following error in my console:

Error: The module '\\?\C:\Users\t-aidaym\work\vscode\node_modules\spdlog\build\Release\spdlog.node' was compiled against a different Node.js version using NODE_MODULE_VERSION 103. This version of Node.js requires NODE_MODULE_VERSION 106

And by searching online, some answers suggested to remove spdlog and reinstall it, which increased it to the latest version.

Reply:
I see all the checks seem to be failing now, I'll revert back to the older version and investigate further the source of my problem.

Reply:
I'll try git clean as you suggested.